docker-compose.yml

```diff
@@ -54,12 +68,6 @@ services:
             - 'matching_data:/data/db'
         networks:
             - backend-network
```

Comment:
Now that the health check is removed, is there still a health check going on somewhere else?

Reply:
nope, but the depends on condition is still valid, it just wont ping the dependency constantly